webknossos/examples/learned_segmenter.py

```diff
@@ -6,6 +6,7 @@
 from skimage.future import TrainableSegmenter
 
 import webknossos as wk
+from webknossos.client._upload_dataset import LayerToLink
```

Comment:
Maybe we should re-import `LayerToLink` at some higher level, e.g. on the `client/__init__.py`, then it's also automatically re-exported toplevel, so it can be used as `wk.LayerToLink`? What do you think?